rero_ils/modules/fees/api_views.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is normally called `views.py`.

Reply:
Done